### ISSUE TITLE
[Request] [Twitch] Left side chat

### ISSUE THREAD

**Issue by dbCoMPy** [SevenTV/Extension#451]

**Comment by gitHabbe** [comment]
Yes, please add this. It makes much more sense to have the chat on the left side if the second monitor is on the right side of the main monitor.

**Comment by SpriggsyUK** [comment]
Yeah this would be incredibly helpful to have nativly within 7tv
